aten/src/ATen/gen.py

```diff
@@ -23,6 +25,11 @@
 
 parser = argparse.ArgumentParser(description='Generate ATen source files')
 parser.add_argument('files', help='cwrap files', nargs='+')
+
+# The Buck-based build system used internally at Facebook passes
+# install_dir rather than output_dir. Accept this variant gracefully.
+parser.add_argument('-i', '--install_dir', help='for use with facebook build infrastructure')
```

Comment:
rename output-dir to install_dir on the OSS side and remove references to special-casing this for facebook infra.